To build in Nonstop

roshanantonyhpe opened the pull request:
changes
1. common directory renamed to +common so that it build first
2. changes in the Makefile to look for the  nsfti  headers and library locations
3. changes in Makefile to look for zsysdefs and zguard headers
4. removed the executable from 03_nsfti_namespace/bin
5. update in 04_dollar_recv_ipc sources

roshanantonyhpe committed:
To build in Nonstop

changes
1. common directory renamed to +common so that it build first
2. changes in the Makefile to look for the  nsfti  headers and library locations
3. changes in Makefile to look for zsysdefs and zguard headers
4. removed the executable from 03_nsfti_namespace/bin
5. update in 04_dollar_recv_ipc sources

SuveerN reviewed (changes requested):
Rename the +common folder as 00_common so that the naming is inline with the rest of the folders. +common does not seem to be intuitive

roshanantonyhpe committed:
To address the pull request comment

1. Updated the directory name to match with the naming convention.
2. Updated the Makefile to look for the updated directory name

SuveerN reviewed (approved):
The changes themselves look good. But the 'commit' comment still has the '+common' in it.
